database/seeders/UserTableSeeder.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Database\Seeders;
+
+use Illuminate\Database\Seeder;
+use Illuminate\Support\Facades\Hash;
+
+class UserTableSeeder extends Seeder
+{
+    public function run(): void
+    {
+        User::truncate();
+
+        $faker = \Faker\Factory::create();
+
+        User::create([
+            "name" => "Admin",
+            "email" => "admin@test.com",
+            "password" => Hash::make("admin123"),
```

Comment:
I would prefer standard value like `password`.
```suggestion
            "password" => Hash::make("password"),
```